.edm4hep-ci.d/compile_and_test_with_podio.sh

```diff
@@ -4,15 +4,16 @@ git clone --depth 1 https://github.com/AIDASoft/podio || true
 cd podio
 mkdir build install
 cd build
-cmake -DCMAKE_CXX_STANDARD=17 -DBUILD_TESTING=OFF -DCMAKE_INSTALL_PREFIX=../install -G Ninja .. \
+cmake -DCMAKE_CXX_STANDARD=17 -DBUILD_TESTING=OFF -DCMAKE_INSTALL_PREFIX=$PWD/../install -DENABLE_SIO=ON -G Ninja .. \
       && ninja -k0 \
       && ninja install \
           || exit 1
 
 cd ..
+# Place this installation in front of any other which might be present in the
+# setup
+unset CPATH
```

Comment:
if you add the CPATH to ROOT_INCLUDE_PATH before unsetting CPATH, the ddsim test should pass
```
export ROOT_INCLUDE_PATH=$CPATH:$ROOT_INCLUDE_PATH
unset CPATH
```
or something